src/TransitionConfigs/CardStyleInterpolators.tsx

```diff
@@ -195,3 +195,22 @@ export function forRevealFromBottomAndroid({
     overlayStyle: { opacity: overlayOpacity },
   };
 }
+
+export function forVerticalInvertedIOS({
```

Comment:
Maybe we don't need a default preset for this

Reply:
I think it's fine.

Reply:
idk if we need to add presets for every option

Reply:
Why not? What would you suggest?

Reply:
Just wanna keep default configs as sane defaults. For custom use cases, users can write their own.

Reply:
🤷‍♂

Reply:
OK, I removed it from default presets and put straight into example